src/server/server.ts

```diff
@@ -317,6 +318,7 @@ function compileCommon(req: express.Request, res: express.Response, wrapWithRunn
             }
 
             timer.stop();
+            res.setHeader('Cache-Control', 'no-cache, no-store');
```

Comment:
You need to add cache-control headers to the error page as well.
    const html = await generateErrorHtml(error, strings);
    return res.contentType('text/html').status(200).send(html);